cartoframes/client/sql_client.py

```diff
@@ -0,0 +1,206 @@
+from __future__ import absolute_import
+
+from carto.exceptions import CartoException
+
+from . import internal
+
+
+class SQLClient(object):
+    """SQLClient class is a client to run SQL queries in a CARTO account.
+
+    Args:
+        creds (:py:class:`Credentials <cartoframes.auth.Credentials>`):
```

Comment:
```suggestion
        credentials (:py:class:`Credentials <cartoframes.auth.Credentials>`):
```

Reply:
I'm proposing in other issues using `credentials` instead of `creds` and in this case, the name of the argument is actually `credentials`